client/signup/steps/woocommerce-install/transfer/transfer-site.tsx

```diff
@@ -42,7 +45,7 @@ export default function TransferSite( {
 	const softwareApplied = softwareStatus?.applied;
 
 	// Check for error codes (5xx). 404's are not a failure mode.
-	const transferFailed =
+	const isTransferringStatusFailed =
```

Comment:
`isTransferringStatusFailed` changes constantly, depending on the transfer and software status. Meaning that it can fail when creating the transferring item, but maybe it will be ok when polling the status.